loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -702,7 +702,11 @@ export class GLTFLoader implements IGLTFLoader {
 
         return Promise.all(promises).then(() => {
             this._forEachPrimitive(node, (babylonMesh) => {
-                babylonMesh.refreshBoundingInfo(true);
+                if ((babylonMesh as Mesh).hasThinInstances) {
+                    // refresh already done
+                } else {
+                    babylonMesh.refreshBoundingInfo(true);
+                }
```

Comment:
See comment in the extension. I don't think you should switch here.

Reply:
Ok, removed.